### PR TITLE
New configuration steps for EPSON

### PR DESCRIPTION
**Pull request by pszafer**
## Proposed change

Update of documentation according to PR https://github.com/home-assistant/core/pull/45434

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45434
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/epson.markdown [hunk 1]
```diff
@@ -31,11 +34,6 @@ host:
   description: The host name or address of the Epson projector
   required: true
   type: string
-port:
```
Comment (frenck) [inline source/_integrations/epson.markdown]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/epson.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_integrations/epson.markdown]: @pszafer I suggest making an additional core PR where we deprecate the configuration via configuration.yaml. Then we can remove the section here completely.

Comment (pszafer) [reply source/_integrations/epson.markdown]: I agree @MartinHjelmare, so what would be the steps?
PR with logger info with deprecation notice and PR for eg. 2021.7 which removes configuration.yaml?

Comment (MartinHjelmare) [reply source/_integrations/epson.markdown]: Log a warning in `async_setup_platform` which will go out in 2021.6 and then remove configuration.yaml support in 2021.8.

**Review by frenck — COMMENTED** [review]
@pszafer Could you update this PR to reflect the changes made in the merged parent PR?

Thanks! 👍

File: source/_integrations/epson.markdown [hunk 2]
```diff
@@ -14,6 +14,8 @@ ha_platforms:
 ---
 
 The `epson` platform allows you to control a Epson projector from Home Assistant.
+**Device has to be turned on during initial configuration.**
+When you want to add device turn it on for first time before following integration steps.
```
Comment (MartinHjelmare) [inline source/_integrations/epson.markdown]:
```suggestion
When you want to add a device for the first time, turn it on before following the integration steps.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_integrations/epson.markdown (hunk 1 above)
Comment (pszafer) [reply source/_integrations/epson.markdown]: @MartinHjelmare  PR with warning added: https://github.com/home-assistant/core/pull/50403

**Review by frenck — APPROVED** [review]

**Pull request merged**